Description: disable installation by setup.py of init script and unit file installation
 policyd-rate-limit.service and policyd-rate-limit.init are identical except for
 /usr/local/bin/ replaced by /usr/bin/. It also let dh_installdeb automatically
 setup symbolic links in rc* in postinst and postrm.
Author: Valentin Samir <valentin.samir@crans.org>

---

Last-Update: <2016-07-09>

--- policyd-rate-limit-0.4.1.orig/setup.py
+++ policyd-rate-limit-0.4.1/setup.py
@@ -23,12 +23,6 @@ def add_data_file(dir, file, check_dir=F
 # if install as root populate /etc
 if os.getuid() == 0:
     add_data_file("/etc", 'policyd_rate_limit/policyd-rate-limit.conf')
-    add_data_file('/etc/init.d', 'init/policyd-rate-limit')
-    add_data_file(
-        "/etc/systemd/system",
-        'init/policyd-rate-limit.service',
-        check_dir=True
-    )
 # else use user .config dir
 else:
     conf_dir = os.path.expanduser("~/.config/")
